### ISSUE TITLE
feature: add language to body-tag

### ISSUE THREAD

**Issue by swidbert** [simonhaenisch/md-to-pdf#274]
**Problem:**

To be able to use hyphenation correctly in the German language, the "html" tag requires the "lang" attribute. The "lang" attribute determines which language the element content uses.

**Solution:**

To do this, simply adjust line 9 in the "get-html.js" file:
```<html${config.body_lang ? ` lang="${config.body_lang}"`: ''}>`````

To activate this in HTML files, this would then be possible by passing the parameter ```body_lang: 'de'``` in the options.

**Comment by simonhaenisch** [comment]
Ok sure do you want to create a PR? I won't have time for it any time soon.

- make the change you suggested in [get-html.ts](https://github.com/simonhaenisch/md-to-pdf/blob/master/src/lib/get-html.ts)
- add the option in [config.ts](https://github.com/simonhaenisch/md-to-pdf/blob/master/src/lib/config.ts) into `BaseConfig` interface and `defaultConfig` object
- add it in [cli.ts](https://github.com/simonhaenisch/md-to-pdf/blob/master/src/cli.ts#L33) after `--body-class` as a CLI flag
- add it in [help.ts](https://github.com/simonhaenisch/md-to-pdf/blob/master/src/lib/help.ts#L16) after `--body-class` with a description
- update the readme (usage and options)

**Comment by swidbert** [comment]
No, that's not the problem. HTML expects the language code to be able to
use hyphenation correctly.

Am Mo., 27. Nov. 2023 um 13:48 Uhr schrieb Simon Hänisch <
***@***.***>:
